test/suites/upgrade.sh

```diff
@@ -150,6 +150,7 @@ ovn:
     # Fourth upgrade MicroCloud.
     for m in micro01 micro02 micro03; do
       lxc exec "${m}" -- snap refresh microcloud --channel "${microcloud_target}"
+      set_debug_binaries "${m}"
```

Comment:
Aren't those already copied to the respective location? Even though MicroCloud 1 snap is ignoring them the binaries should still have been copied to the `/var/snap/microcloud/common/` path and AFAIK the `snap refresh` isn't modifying this.

Reply:
Should get set [here](https://github.com/canonical/microcloud/blob/main/test/includes/microcloud.sh#L1277) already.

Reply:
Yeah good point. Maybe adding a condition in the `setup_system` to only inject if MicroCloud >= 2? Same as with the `dm-crypt` stuff.

Reply:
Agreed, except instead of >=2 it would be only inject if `/edge`, and skip it `/stable`.

Reply:
Actually now that I'm thinking about it, silently having cases that skip the debug binary can be very destructive if you forget about it. 

Might just be better to Manually delete the debug binaries in the upgrade tests.